components/infobox/wikis/wildrift/infobox_team_custom.lua

```diff
@@ -0,0 +1,90 @@
+---
+-- @Liquipedia
+-- wiki=wildrift
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Template = require('Module:Template')
+local Variables = require('Module:Variables')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local Team = Lua.import('Module:Infobox/Team', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+
+local CustomTeam = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _args
+local _team
+
+function CustomTeam.run(frame)
+	local team = Team(frame)
+	_team = team
+	_args = _team.args
+
+	-- Automatic org people
+	team.args.coach = Template.expandTemplate(frame, 'Coach of')
+	team.args.manager = Template.expandTemplate(frame, 'Manager of')
+	team.args.captain = Template.expandTemplate(frame, 'Captain of')
+
+	team.createWidgetInjector = CustomTeam.createWidgetInjector
+	team.createBottomContent = CustomTeam.createBottomContent
+	team.addToLpdb = CustomTeam.addToLpdb
+	team.getWikiCategories = CustomTeam.getWikiCategories
+	return team:createInfobox(frame)
+end
+
+function CustomTeam:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomTeam:createBottomContent()
+	if not _team.args.disbanded and mw.ext.TeamTemplate.teamexists(_team.pagename) then
+		local teamPage = mw.ext.TeamTemplate.teampage(_team.pagename)
+
+		return Template.expandTemplate(
+			mw.getCurrentFrame(),
+			'Upcoming and ongoing matches of',
+			{team = _team.lpdbname or teamPage}
+		) .. Template.expandTemplate(
+			mw.getCurrentFrame(),
+			'Upcoming and ongoing tournaments of',
+			{team = _team.lpdbname or teamPage}
+		) .. Template.expandTemplate(
+			mw.getCurrentFrame(),
+			'Placement summary',
+			{team = _team.lpdbname or teamPage}
+		)
+	end
+end
```

Comment:
```suggestion
function CustomTeam:createBottomContent()
	if not _team.args.disbanded then
		return Template.expandTemplate(
			mw.getCurrentFrame(),
			'Upcoming and ongoing matches of'
		) .. Template.expandTemplate(
			mw.getCurrentFrame(),
			'Upcoming and ongoing tournaments of'
		) .. Template.expandTemplate(
			mw.getCurrentFrame(),
			'Placement summary'
		)
	end
end
```

Verify that this works on
* Team with Space
* Team with lowercase first letter

Reply:
> Verify that this works on
> 
> * Team with Space
> * Team with lowercase first letter

Done, It does disappear for disbanded teams 

Active with Space: https://liquipedia.net/wildrift/Keyd_Stars
Disbanded with Space: https://liquipedia.net/wildrift/Bigetron_Infinity

Disbanded with both Space and lowercase start (since this is the only lowercase start team I could find in the wiki): https://liquipedia.net/wildrift/Ahq_e-Sports_Club